src/renderer/components/Map.tsx

```diff
@@ -118,9 +107,9 @@ function makeStationMarker(
     </>
   );
 
-  const marker = new mapboxgl.Marker(markerElement)
+  const marker = new mapboxgl.Marker(markerElement, { anchor: 'top' })
```

Comment:
The anchor is the part of the marker which will be placed at the coordinates; we need anchor `bottom` here. I guess you did it to avoid the overlap with the popup. We need to offset its position instead.

Reply:
Done